src/LibraryManager.Vsix/UI/InstallDialogTestContract.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI
+{
+    /// <summary>
+    /// This class gives apex access to opened - add client side libraries dialog.
+    /// </summary>
+    internal class InstallDialogTestContract
```

Comment:
>InstallDialogTestContract [](start = 19, length = 25)

Perhaps part of what bugs me is the naming of this class. I seems to be a provider of an IInstallDialogTestContract, not an implementation of that contract.

Reply:
Updated in iteration 5

---
In reply to: [205942398](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205942398) [](ancestors = 205942398)